Util/FileUtils.php

```diff
@@ -51,12 +53,37 @@ public static function findTranslationFiles($directory)
                 $file
             );
         }
-        
+
         uksort($files, 'strcasecmp');
-        
+
         return $files;
     }
 
+    /**
+     * @param string $path
+     * @param string $rootPath
+     *
+     * @return string
+     */
+    public static function getRelativePath($path, $rootPath)
```

Comment:
Lets put this function in the FileSource factory until it is needed on other places. 
Also, make it private.